self-tests.yml

```diff
@@ -585,6 +596,22 @@ tasks:
             LOCAL_PATH: clients
             EXCLUDE_PATTERN: .cache
             REMOTE_PATH: evergreen/clients/${project}/${revision}
+  - name: swaggo-lint
```

Comment:
nit: make it consistent with the other linters (e.g. `verify-swaggo-fmt`) + attach the `tags: ["linter"]` so that it gets automatically picked up with the other linter-tagged tasks on L785.